lib/features/authentication/login/page/login.dart

```diff
@@ -50,33 +47,33 @@ class __LoginPageViewState extends State<_LoginPageView> {
         ),
         //
         Padding(
-          padding: EdgeInsets.only(left: 25.w, right: 25.w),
+          padding: EdgeInsets.only(top: 77.h, left: 25.w, right: 25.w),
           child: SizedBox(
-            height: 1.sh,
-            width: 1.sw,
+            height: double.infinity.sh,
+            width: double.infinity.sw,
             child: SingleChildScrollView(
               child: Form(
                 child: Column(
                   children: [
                     SizedBox(
+                      height: 88.h,
                       width: 390.w,
                       child: Column(
                         mainAxisAlignment: MainAxisAlignment.spaceAround,
                         children: [
                           Text(
                             'Login here',
                             style: TextStyle(
-                              color: AppColors.color491702,
+                              color: const Color(0xff491702),
```

Comment:
revert back to use colors from static properties of AppColors
                              color: AppColors.color491702,